latex/ledger-spec.tex

```diff
@@ -224,15 +227,30 @@ \section{Notation}\label{sec:notation}
 \section{Cryptographic primitives}
 \label{sec:crypto-primitives}
 
+
 Figure~\ref{fig:crypto-defs} introduces the cryptographic abstractions used in
-this document.
+this document. We begin by listing the abstract types, which are meant to
+represent the corresponding concepts in cryptography. Only the funcitonality
+explicitly stated in the figures below is assumed within the scope of this paper.
+That is, their exact
+implementation remains open to interpretation, and we do not rely on
+any additional properties derived from the study or implementation of public key
+cryptography outside this work. The types and rules we give here are needed in
+order to guarantee certain security properties of the delegation process, which
+we discuss later.
+
+The cryptographic concepts required for the formal definition
+of delegation with witnesses include public-private key pairs, hash
+maps and signatures. The constraint we introduce states that a signature of
```

Comment:
were we going to say "one-way functions" instead of hash maps?

Reply:
Yes!